pkg/snapshotmgr/snapshot_manager.go

```diff
@@ -104,8 +105,7 @@ func NewSnapshotManagerFromCluster(params map[string]interface{}, config map[str
 		logger.Infof("SnapshotManager: Get s3PETM from the params map")
 	}
 
-	// If the local mode is enabled, we don't need to specify remote storage location
-	// for persistence since it has been taken care of 3rd party backup solution.
+	// Initializing IVD ProtectedEntityTypeManager
```

Comment:
Remove it. No need to init IVD PETM since Direct PETM will be initialized.

Reply:
same as above.